### PR TITLE
[PHP 8.4] `odbc_fetch_row()` can emit a `E_WARNING`

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Part of #3872

### CODE REVIEW

File: reference/uodbc/functions/odbc-fetch-row.xml [hunk 1]
```diff
@@ -72,6 +72,13 @@
      </row>
     </thead>
     <tbody>
+     <row>
+      <entry>8.4.0</entry>
+      <entry>
+       An <constant>E_WARNING</constant> is now emitted when <parameter>row</parameter>
+       is equal or less than zero.
+      </entry>
+     </row>
```
Comment (Girgias) [inline reference/uodbc/functions/odbc-fetch-row.xml:81]: This is missing an error section after the return value section.

Comment (alexandre-daubois) [reply reference/uodbc/functions/odbc-fetch-row.xml:89]: Section added 👍

**Review by Girgias — APPROVED** [review]

File: reference/uodbc/functions/odbc-fetch-row.xml [hunk 2]
```diff
@@ -61,6 +61,14 @@
...
  </refsect1>
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <simpara>
+   An <constant>E_WARNING</constant> is emitted when <parameter>row</parameter>
+   is equal or less than zero.
+  </simpara>
+ </refsect1>
+
```
Comment (Girgias) [inline reference/uodbc/functions/odbc-fetch-row.xml:71]:
```suggestion

```

File: reference/uodbc/functions/odbc-fetch-row.xml [hunk 3]
```diff
@@ -61,6 +61,13 @@
...
  </refsect1>
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <simpara>
+   An <constant>E_WARNING</constant> is emitted when <parameter>row</parameter>
+   is equal or less than zero.
+  </simpara>
+ </refsect1>
  <refsect1 role="changelog">
```
Comment (Girgias) [inline reference/uodbc/functions/odbc-fetch-row.xml:71]:
```suggestion
 </refsect1>

 <refsect1 role="changelog">
```

**Pull request merged**